README.md

```diff
@@ -42,25 +42,15 @@ Below are helpful resources for Fluo application developers:
 *  [Fluo Recipes][recipes] is a project that provides common code for Fluo application developers
    implemented using the Fluo API.
 
-## Implementation
-
-*  [Architecture] - Overview of Fluo's architecture
-*  [Contributing] - Documentation for developers who want to contribute to Fluo
-*  [Metrics] - Fluo metrics are visible via JMX by default but can be configured to send to Graphite
-   or Ganglia
-
 [fluo]: https://fluo.apache.org/
 [related]: https://fluo.apache.org/related-projects/
 [tour]: https://fluo.apache.org/tour/
 [accumulo]: https://accumulo.apache.org
 [percolator]: https://research.google.com/pubs/pub36726.html
-[install]: docs/install.md
-[apps]: docs/applications.md
+[fluo-docs]: https://fluo.apache.org/docs/fluo/1.1.0-incubating/
```

Comment:
What about linking to `https://fluo.apache.org/docs/` instead of a specific version?

Reply:
Good idea.